docker-for-mac/networking.md

```diff
@@ -39,9 +39,30 @@ $ docker run -p 8000:80 -d nginx
 Now, connections to `localhost:8000` are sent to port 80 in the container. The
 syntax for `-p` is `HOST_PORT:CLIENT_PORT`.
 
+### SSH agent forwarding
+
+Docker Desktop for {{Arch}} allows you to use the host’s SSH agent inside a container. To do this:
+
+1. Bind mount the SSH agent socket by adding the following parameter to your `docker run` command:
+
+    `-v /run/host-services/ssh-auth.sock:/run/host-services/ssh-auth.sock`
+
+1. Add the `SSH_AUTH_SOCK` environment variable in your container:
+
+    `-e SSH_AUTH_SOCK="/run/host-services/ssh-auth.sock"`
+
+To enable the SSH agent in Docker Compose, add the following flags to your service:
+
+ ```
+    volumes:
```

Comment:
Given that `volumes:` can either be at the top-level (to _define_ a volume) or under a service (to _use_ a volume), perhaps we should show a more complete example (with `services:`)?

    ```yaml
    services:
      web:
        image: nginx:alpine
        volumes:
          - type: bind
            source: /run/host-services/ssh-auth.sock
            target: /run/host-services/ssh-auth.sock
        environment:
          - SSH_AUTH_SOCK=/run/host-services/ssh-auth.sock
    ```

Reply:
Thanks, it's a better example!

Reply:
Updated the example